app/src/marketplace/containers/ProjectEditing/DeleteProject.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import styled from 'styled-components'
+import Button from '$shared/components/Button'
+import { useProjectController } from "./ProjectController"
+
+const Title = styled.p`
+  font-size: 34px;
+  line-height: 34px;
+  color: black;
+  margin-bottom: 30px;
+`
+
+const Description = styled.p`
+  color: black;
+  font-size: 16px;
+  padding-bottom: 28px;
+`
+
+const DeleteProject = () => {
+    const { deleteProject } = useProjectController()
+
+    return (
+        <div>
+            <Title>Delete project</Title>
+            <Description>Delete this project forever. You can&apos;t undo this.</Description>
+            <Button
+                kind='destructive'
+                onClick={async () => { deleteProject() }}
```

Comment:
`async` doesn't do anything here (no `await`). It's the same as `onClick={() => void deleteProject()}`.

With that said, it does not take care of any exceptions that come from within `deleteProject`. It'd even be better to just log it.

```tsx
onClick={async () => {
    try {
        await deleteProject()
    } catch (e) {
        console.warn('Could not delete a project', e)
    }
}}
```